### ISSUE TITLE
Found tab character (`\t`) in file. for tabs in a literal string

### ISSUE THREAD

**Issue by smoothdeveloper** [fsprojects/FSharpLint#148]
IMHO this shouldn't trigger this lint warning:
```fsharp
let a = @"a	b"
let b = """
a	b
"""
```

(those are tabs but github replace with spaces)

**Comment by duckmatt** [comment]
Thanks for the report, I agree that's a bug in that it reports a warning there

**Comment by smoothdeveloper** [comment]
@duckmatt / @mexx great, thanks for quick turnaround!
